admin/.env

```diff
@@ -0,0 +1,2 @@
+VITE_BACKEND_URL=http://localhost:5000
+/admin
```

Comment:
:scream: 
WHY IS THERE A .ENV FILE HERE ?!
@LuigiMenez ! SOMEONE!!! ALAAAAARM !!!!!!! :skull: :warning: